codegenerator/cli/templates/static/codegen/src/eventFetching/chainWorkers/HyperFuelWorker.res

```diff
@@ -264,29 +264,6 @@ module Make = (
     )
   }
 
-  let waitForNextBlockBeforeQuery = async (
-    ~serverUrl,
-    ~fromBlock,
-    ~currentBlockHeight,
-    ~logger,
-    ~setCurrentBlockHeight,
-  ) => {
-    if fromBlock > currentBlockHeight {
-      logger->Logging.childTrace("Worker is caught up, awaiting new blocks")
-
-      //If the block we want to query from is greater than the current height,
-      //poll for until the archive height is greater than the from block and set
-      //current height to the new height
-      let currentBlockHeight = await HyperFuel.pollForHeightGtOrEq(
-        ~serverUrl,
-        ~blockNumber=fromBlock,
-        ~logger,
-      )
-
-      setCurrentBlockHeight(currentBlockHeight)
-    }
-  }
```

Comment:
Can safely remove it now, because `fetchBlockRange` will never be called with invalid fromBlock/toBlock